.github/workflows/wheels.yml

```diff
@@ -1,49 +1,63 @@
-# This file is autogenerated by maturin v1.7.4
-# To update, run
-#
-#    maturin generate-ci github
-#
-name: wheels
+name: Build wheels
 
 on:
   push:
     tags:
-      - "py-v*"
+      - "*"
+  pull_request:
+    branches: [main]
   workflow_dispatch:
 
 permissions:
   contents: read
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
 jobs:
   linux:
     runs-on: ${{ matrix.platform.runner }}
     strategy:
+      fail-fast: false
+
       matrix:
         platform:
           - runner: ubuntu-latest
             target: x86_64
-          - runner: ubuntu-latest
-            target: x86
+            manylinux: auto
+          # TODO: gcc can't find a suitable -lgcc on 32-bit linux
+          # - runner: ubuntu-latest
+          #   target: x86
+          #   manylinux: auto
           - runner: ubuntu-latest
             target: aarch64
+            manylinux: auto
           - runner: ubuntu-latest
             target: armv7
+            manylinux: auto
           - runner: ubuntu-latest
             target: s390x
+            manylinux: auto
           - runner: ubuntu-latest
             target: ppc64le
+            manylinux: auto
     steps:
       - uses: actions/checkout@v4
-      - uses: actions/setup-python@v5
-        with:
-          python-version: 3.x
+
       - name: Build wheels
         uses: PyO3/maturin-action@v1
         with:
           target: ${{ matrix.platform.target }}
-          args: --release --out dist --find-interpreter
+          args: --release --out dist -i 3.10 -i 3.11 -i 3.12 -i 3.13
           sccache: "true"
-          manylinux: auto
+          manylinux: ${{ matrix.manylinux }}
+          before-script-linux: |
+            sudo apt-get update
+            sudo apt-get install -y libproj-dev sqlite3 libsqlite3-dev pkg-config
+
+            export PKG_CONFIG_ALLOW_CROSS=1
```

Comment:
If using `pkg-config`, maybe you need to enable the `pkg_config` feature flag on `proj-sys` as mentioned at https://docs.rs/proj/0.28.0/proj/#feature-flags?

Reply:
that doesn't help, unfortunately. I _think_ this would succeed if I somehow managed to get a cross-compiled `libproj` (which I guess is why the x86_64 build succeeds? No clue why the macos builds succeed).